### DISCUSSION TITLE
sky mod

### DISCUSSION THREAD

**Discussion by moha1218** [EnderiteFox/Friday-night-funkin-with-Pygame#12] [Ported mods]
sky mod ported to fnf pygame but with black screen because a bug but it work correctly
link: https://www.mediafire.com/file/rh9rjcn19c346yx/sky.zip/file

**Comment by EnderiteFox** [comment]
I've tested it, and everything works fine. You could also fix the second song, where the "-" character isn't supported by the font used for the game

**Comment by moha1218** [comment]
ok thanks
